timedctl/timedctl.py

```diff
@@ -463,8 +507,43 @@ def show():
         error_handler("ERR_NO_CURRENT_ACTIVITY")
 
 
-@activity.command(aliases=["gts", "ts"])
-def generate_timesheet():
+@activity.command("restart", aliases=["r", "continue", "resume"])
+@click.option("--date", default=None)
+def restart_activity(date):
+    """Restart an activity."""
+    # stop current activity first
+    if timed.activities.current:
+        timed.activities.stop()
+        msg("Stopped current activity.")
+    # select an activity
+    activity = select_activity(date)
+    # grab attributes
+    comment = activity[1]
+    task_id = activity[3]
+    res = timed.activities.start(
+        attributes={"comment": comment}, relationships={"task": task_id}
```

Comment:
Why does the comment need to be added as a param if the activity already exists?

Reply:
So you are not restarting the activity but creating a new one?